### PR TITLE
add: views/unitconverter.xaml

### PR DESCRIPTION
**Pull request by tian-lt**
### Description of the changes:
- migrates UnitConverter from C++/CX to C#
- modifies some interface of UnitConverterViewModel

### How changes were validated:

- Build pass
- Unit Tests and UI Tests are not applicable at current stage.

### CODE REVIEW

**Comment by tian-lt** [comment]
Hi @hanzhang54,
Could you help review this PR by the end of tomorrow? thx!

File: src/Calculator/Views/MainPage.xaml.cs [hunk 1]
```diff
@@ -38,6 +38,7 @@ public MainPage()
             m_model.PropertyChanged += OnAppPropertyChanged;
             m_accessibilitySettings = new AccessibilitySettings();
 
+
```
Comment (hanzhang54) [inline src/Calculator/Views/MainPage.xaml.cs]: Please remove the extra empty lines here and in UnitConverter.xaml.cs

Comment (tian-lt) [reply src/Calculator/Views/MainPage.xaml.cs]: done

**Review by hanzhang54 — APPROVED** [review]
LGTM

**Pull request merged**